core/src/main/java/org/apache/accumulo/core/security/crypto/EncryptionStrategy.java

```diff
@@ -0,0 +1,30 @@
+package org.apache.accumulo.core.security.crypto;
+
+import java.io.InputStream;
+import java.io.OutputStream;
+import java.util.HashMap;
+import java.util.Map;
+
+import org.apache.accumulo.core.conf.AccumuloConfiguration;
+import org.apache.accumulo.core.conf.Property;
+
+public interface EncryptionStrategy {
+
+  public void encryptStream(OutputStream outputStream);
+
+  public void decryptStream(InputStream inputStream);
+
+  public Map<String,String> properties = new HashMap<>();
```

Comment:
These properties shouldn't be public, and the getters/setters shouldn't have default methods.